embeddings/data/datamodule.py

```diff
@@ -261,9 +261,7 @@ def _class_encode_column(self, column_name: str) -> None:
             self.dataset = self.dataset.class_encode_column(column_name)
         else:
             new_features = self.dataset["train"].features.copy()
-            new_features[column_name] = ClassLabel(
```

Comment:
why skipping names?

Reply:
Because in new version of `transformers` library they skipped this parameter. I will update requirements to the latest to consider latest one.